### DISCUSSION TITLE
react-aria: drag item to a reorderable list example

### DISCUSSION THREAD

**Discussion by fredericrous** [adobe/react-spectrum#6193] [Q&A] [answered]
with react-aria, I would like to drag an item from a list to another list and be able to reorder it in the new list

the closer example I could find is on the spectrum storybook bit it does not use totally react-aria:
 http://localhost:9003/?path=/story/listview-drag-and-drop--drag-between&providerSwitcher-express=false&strict=true with react-aria?

I tried a few things, spent a good amount of time on it but I didn't get it. Could someone please share an example?

**Comment by snowystinger** [comment, marked as answer]
https://react-spectrum.adobe.com/react-aria/ListBox.html#drag-between-lists

**Reply by fredericrous** [reply]
I thought useDragAndDrop was only part of spectrum! I see it is now part of react-aria. I'll try again. thank you

**Reply by fredericrous** [reply]
I succeeded implementing the component. long live react-aria-components 🎉
